autoload/zcd.vim

```diff
@@ -76,9 +76,15 @@ func! s:GetSearchOutput(search) abort
     return v:null
   endif
 
-  " source z.sh; _z -l 'some search term'
-  let l:cmd = 'source ' . fnameescape(l:z_path)
-  let l:cmd .= '; _z -l ' . shellescape(a:search)
+  " source z.sh(z.lua); _z(_zlua) -l 'some search term'
+  if l:z_path =~ 'z.lua'
+    let l:cmd = 'eval "$(lua '. fnameescape(l:z_path).' --init bash)"'
+    let l:cmd .=';export _ZL_HYPHEN=1'
+    let l:cmd .= '; _zlua -l ' . shellescape(a:search)
+  elseif l:z_path =~ 'z.sh'
```

Comment:
If the `l:z_path` isn't named `z.sh` but still points to a valid script, `l:cmd` won't get set and it might go :boom:  
Would you switch this to an `else` catchall?